### PR TITLE
Http only toggle

### PR DESCRIPTION
**Pull request by vriken**
Had a conversation with @Bukashk0zzz on the measure slack, creating this PR thereafter

https://measure.slack.com/archives/C05EQBHEJUS/p1733929018672439?thread_ts=1733926783.340039&cid=C05EQBHEJUS

### CODE REVIEW

**Comment by vriken** [comment]
I see that this contains two commits from our fork, which shouldnt be here, I'll remove them

**Review by Bukashk0zzz — CHANGES_REQUESTED** [review]
Thanks for the PR

**Comment by vriken** [comment]
> Thanks for the PR

Added the OR false @Bukashk0zzz

File: template.tpl [hunk 1]
```diff
@@ -36,6 +36,22 @@ ___TEMPLATE_PARAMETERS___
     "simpleValueType": true,
     "help": "If enabled, the server only accessible FPID cookie, generated by UA/GA4 client, will be duplicated to FPIDP cookie, which will be accessible from the client JS. Highly recommend using this option only in case it is necessary."
   },
+  {
```
Comment (Bukashk0zzz) [inline template.tpl:39]: @vriken now here is duplicate.

**Review by Bukashk0zzz — CHANGES_REQUESTED** [review]

File: template.tpl (hunk 1 above)
Comment (vriken) [reply template.tpl:39]: Getting real messy with my commits :s. Sorry about that

Comment (vriken) [reply template.tpl:39]: Should be fixed now

**Review by Bukashk0zzz — APPROVED** [review]

**Pull request merged**